multinet/uploaders/csv.py

```diff
@@ -111,10 +122,13 @@ def upload(workspace: str, table: str) -> Any:
     # Read the request body into CSV format
     body = decode_data(request.data)
 
-    rows = list(csv.DictReader(StringIO(body)))
+    # Type to a Dict rather than an OrderedDict
+    rows: List[Dict[str, str]] = list(csv.DictReader(StringIO(body)))
 
     # Perform validation.
-    validate_csv(rows)
+    key = key or "_key"
```

Comment:
Instead of this, why not just let the default arg value be `"_key"`? Then you could also type the `key` arg as `str` instead of `Optional[str]`.

Reply:
Good point. I can change that.